refinery/tool_manager/urls.py

```diff
@@ -11,7 +13,14 @@
 tool_manager_router.register(r'tools', ToolsViewSet)
 tool_manager_router.register(r'tool_definitions', ToolDefinitionsViewSet)
 
-url_patterns = Proxy(FileLogger(settings.PROXY_LOG)).url_patterns()
```

Comment:
Could we just pass in a template and have the `PleaseWaitView` render it? Not sure if thats the best approach, but I think it would be more straightforward than this approach. 

This could be shortsighted of me, but it would be easier to extend a template in the future, than a simple string.

Could utilize the nifty:
`django.shortcuts.render_to_response(template)`

Reply:
We talked, and there's really not that much information that would go in the template. If we did want the template to be filled out for each request (perhaps giving information coming from Docker? Or depending on the type of underlying error we hit?) that would need to be done inside of django_docker_engine.